src/EventListener/TimestampEntityChanges.php

```diff
@@ -57,26 +56,26 @@ public function postPersist(LifecycleEventArgs $args)
      */
     protected function stamp(object $entity)
     {
-        $timestamp = new Datetime();
+        $now = DateTime::createFromFormat('U', (string) time());
```

Comment:
the only difference i could spot here is that the newly generated DateTime zeros out milliseconds, whereas the old form carried that level of precision.
which has no practical impact for us.